### ISSUE TITLE
Define static MPDs as always available

### ISSUE THREAD

**Issue by sandersaares** [Dash-Industry-Forum/DASH-IF-IOP#261]
There is some text in IOP about availability times of static MPD segments. This seems unrealistic - I suspect everyone uses static MPDs for static file serving, with effectively infinite availability. I do not think any client is going to consider static MPD availability no matter what IOP says.

We should reflect this in IOP - availability is only something that applies to dynamic MPDs. This is already considered in #210 where timing model is concerned but it also affects the service types, so is a wider aspect in general.

**Comment by technogeek00** [comment]
I would agree with this, when an `MPD@type="static"` the statement is that anything within the MPD can be accessed at any time so there is no reason to attempt the application of availability to segments within the player or the service offering.

**Comment by waqarz** [comment]
> There is some text in IOP about availability times of static MPD segments. This seems unrealistic - I suspect everyone uses static MPDs for static file serving, with effectively infinite availability. I do not think any client is going to consider static MPD availability no matter what IOP says.
> 
> We should reflect this in IOP - availability is only something that applies to dynamic MPDs. This is already considered in #210 where timing model is concerned but it also affects the service types, so is a wider aspect in general.

Can you please cite the exact text or Sections etc. you are referring to?

**Comment by sandersaares** [comment]
I refer to this statement in 3.10.1:

> DASH Media Presentations with MPD@type set to "static" indicate that all media announced in the MPD is available at the MPD@availabilityStartTime and will cease to be available at the MPD@availabilityEndTime.

There may also be other statements - I did not review every mention.

**Comment by haudiobe** [comment]
very simple high-level check. Do it once - then done. We can not reference the feature, but it is still present in MPEG DASH. We can make the window to infinity.
